l10n_fr_rup/models/hr_contract.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2020 Akretion (http://www.akretion.com/)
+
+from odoo import models, fields, api
+
+
+class HrContract(models.Model):
+    _inherit = "hr.contract"
+    _order = "date_start,name"
+
+    pcs_id = fields.Many2one("hr.employee.pcs", "PCS")
+    employer_address_id = fields.Many2one("res.partner", "Addresse de l'employeur")
```

Comment:
address_id and work_location doesn't seems to be required field by law. Or did I missed something ? 
https://www.service-public.fr/professionnels-entreprises/vosdroits/F1784

Reply:
employer_address_id refers to the temporary work contractor ("agence d'intérim") company address.
Official government page doesn't give any example of a RUP. This was made in accordance to our client's RUP, so we presume it is compliant and leave it as it is. If there are other concrete examples we can draw inspiration from maybe we can make some modifications.